gradle/libs.versions.toml

```diff
@@ -25,10 +29,12 @@ micronaut-reactor-http-client = { module = 'io.micronaut.reactor:micronaut-react
 micronaut-runtime = { module = 'io.micronaut:micronaut-runtime' }
 micronaut-tracing = { module = 'io.micronaut:micronaut-tracing' }
 micronaut-validation = { module = 'io.micronaut:micronaut-validation' }
+micronaut-docs-asciidoc-config-props = { module = 'io.micronaut.docs:micronaut-docs-asciidoc-config-props', version.ref = 'micronaut-docs' }
 
+javax-annotation-api = { module = 'javax.annotation:javax.annotation-api' }
```

Comment:
remove this catalog item and dependencies on `javax.annotation`. We are migrating code to use `jakarta.annotation` which mostly affects `@PostCreate` and `@PreDestroy` annotations.

Reply:
Fair enough, OK.  The dependency is exposed by the micronaut catalog, so use that instead of making one in `libs`

`mn.javax.annotation.api`